_delphi_utils_python/pyproject.toml

```diff
@@ -0,0 +1,60 @@
+[build-system]
+requires = ["setuptools", "setuptools-scm>=8.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "delphi-utils"
+version = "0.3.24"
+description = "Shared Utility Functions for Indicators"
+readme = "README.md"
+requires-python = "== 3.8.*"
+license = {text = "MIT License"}
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "Programming Language :: Python :: 3.8",
+    "License :: MIT"
+]
+dependencies = [
+    "boto3",
+    "covidcast",
+    "cvxpy",
+    "epiweeks",
+    "gitpython",
+    "importlib_resources>=1.3",
+    "numpy",
+    "pandas>=1.1.0",
```

Comment:
Do we change the version of this to 2.2.2?

Reply:
Not yet, our pandas was indirectly pinned to <1.5.3 through our dependence on `covidcast` (more info [here](https://github.com/cmu-delphi/covidcast-indicators/issues/1972#issuecomment-2197688725)) and the change to 2.0.0 brought a number of breaking changes, so we should do the Pandas upgrade slowly.